dev/start

```diff
@@ -52,6 +55,10 @@ if [[ $ENABLE_AUTHN_LDAP = true ]]; then
   docker-compose exec conjur conjurctl policy load cucumber /src/conjur-server/dev/files/authn-ldap/policy.yml
 fi
 
+if [[ $ENABLE_AUTHN_LDAP = true ]]; then
+  env_args="$env_args -e CONJUR_AUTHENTICATORS=authn-iam/prod"
```

Comment:
For this to work, we'll need to add a IAM Authenticator policy.  For the Authn-LDAP authenticator, this policy file is located in `dev/files/authn-ldap`.  How about adding an `authn-iam` folder with a corresponding policy file to the `dev/files` directory?

Reply:
Added the root policy file with authn-iam in place